lib/customisation.nix

```diff
@@ -66,28 +66,31 @@ rec {
   */
   makeOverridable = f: origArgs:
     let
-      ff = f origArgs;
+      result = f origArgs;
+
       # Creates a functor with the same arguments as f
       copyArgs = g: lib.setFunctionArgs g (lib.functionArgs f);
```

Comment:
If `f` was already a functor, we now have multiple levels of nested attrsets with essentially the same information, right? Shouldn’t we check for that?

Reply:
Yeah, I just added a new commit that makes `setFunctionArgs` not do unnecessary nesting